ibis/bigquery/tests/test_client.py

```diff
@@ -276,13 +275,15 @@ def test_scalar_param_boolean(alltypes, df):
     ).sort_values('id').reset_index(drop=True)
     expected = df.loc[
         df.string_col.astype('int64') == 0
-    ].sort_values('id').reset_index(drop=True)
+        ].sort_values('id').reset_index(drop=True)
```

Comment:
You shouldn't need to make this change. The lint errors are only on a few lines: https://circleci.com/gh/missing-semicolon/ibis/642?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link